spec/waypoint_spec.js

```diff
@@ -338,10 +338,20 @@ describe('<Waypoint>', function() {
   describe('when the scrollable parent does not have positioning', () => {
     beforeEach(() => {
       delete this.parentStyle.position;
+      this.props.onEnter.calls.reset();
+      this.props.onLeave.calls.reset();
+    });
+
+    it('does not call handlers to begin with', () => {
+      expect(this.subject).not.toThrow();
```

Comment:
You could change this to `this.subject();`, since the throw is a thing of the past.